test/selenium-test/sign-in-widget/app.module.ts

```diff
@@ -0,0 +1,64 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule, Router } from '@angular/router';
+
+import {
+  OKTA_CONFIG,
+  OktaAuthModule,
+  OktaCallbackComponent,
+  OktaAuthGuard
+} from '@okta/okta-angular';
+
+import { AppComponent } from './app.component';
+import { ProtectedComponent } from './protected.component';
+import { LoginComponent } from './login.component';
+
+const config = {
+  issuer: 'https://dev-411042.okta.com/oauth2/default',
```

Comment:
We should take these values from env vars instead of hard coding

Reply:
@vijetmahabaleshwar-okta 
Now it passed as env vars. See my last changes.